bundles/org.openhab.binding.volumio/src/main/java/org/openhab/binding/volumio/internal/VolumioHandler.java

```diff
@@ -0,0 +1,382 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.volumio.internal;
+
+import java.math.BigDecimal;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.openhab.binding.volumio.internal.mapping.VolumioData;
+import org.openhab.binding.volumio.internal.mapping.VolumioEvents;
+import org.openhab.binding.volumio.internal.mapping.VolumioServiceTypes;
+import org.openhab.core.library.types.NextPreviousType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.PercentType;
+import org.openhab.core.library.types.PlayPauseType;
+import org.openhab.core.library.types.RewindFastforwardType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.thing.Channel;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseThingHandler;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.RefreshType;
+import org.openhab.core.types.UnDefType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.socket.client.Socket;
+import io.socket.emitter.Emitter;
+
+/**
+ * The {@link VolumioHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Patrick Sernetz - Initial Contribution
+ * @author Chris Wohlbrecht - Adaption for openHAB 3
+ * @author Michael Loercher - Adaption for openHAB 3
+ */
+@NonNullByDefault
+public class VolumioHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(VolumioHandler.class);
+
+    private @Nullable VolumioService volumio;
```

Comment:
There are many issues with the null handling of this field according to the compiler.
Can you fix them?

```
[WARNING] .../VolumioHandler.java:[97,25] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[105,25] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[113,25] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[121,25] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[128,29] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[137,25] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[143,25] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[148,21] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[176,13] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[194,13] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[196,13] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[204,13] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[216,21] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[219,21] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[226,21] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[229,21] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[241,13] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[254,9] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[255,9] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[256,9] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[285,17] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[296,21] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[299,21] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[304,13] Potential null pointer access: this expression has a '@Nullable' type
[WARNING] .../VolumioHandler.java:[311,14] Potential null pointer access: this expression has a '@Nullable' type
```

Reply:
I am working on that in the next version